### PR TITLE
Kamos invitation letters customized

### PR DESCRIPTION
**Pull request by KAMO333**
Reads a list of names from a text file (invited_names.txt) and a template letter from a .docx file (starting_letter.docx). It then customizes the letter for each name, replacing the placeholder [name] with each individual’s name. Finally, it writes the personalized letters into a new text file (send.txt). The write_to_txt() function triggers the entire process and confirms when the task is completed.

### CODE REVIEW

File: mail merge/main.py
Comment (Kyle-ND) [inline mail merge/main.py:1]: Love the seperation of concerns and the code looks good but i doesn't get the task quite done first i would like you to save each letter to a seprate file with a docx extension and then store it into the ready to send folder

File: mail merge/ready_to_send/send.txt
Comment (Kyle-ND) [inline mail merge/ready_to_send/send.txt:1]: seperate these letters into single letters not one big txt

**Review by Kyle-ND — COMMENTED** [review]
Thank you Kamo make sure to look through the changes

**Comment by KAMO333** [comment]
Okay I get what you saying, will do so

**Pull request closed without merging**